libs/designsystem/src/lib/components/empty-state/empty-state.component.spec.ts

```diff
@@ -35,4 +40,32 @@ describe('EmptyStateComponent', () => {
     const outlineElement = element.getElementsByClassName('icon-outline')[0];
     expect(outlineElement).toHaveComputedStyle({ 'border-width': DesignTokenHelper.size('xxxs') });
   });
+
+  describe('Attention level rules for buttons', () => {
+    let buttons: ButtonComponent[];
+
+    beforeEach(() => {
+      buttons = spectator.queryAll(ButtonComponent);
+    });
+
+    it('should change all buttons to attention level 3 except 1', () => {
+      const numberLvl3Buttons = buttons.filter((button) => button.isAttentionLevel3).length;
+      const numberLvl1Buttons = buttons.filter((button) => button.isAttentionLevel1).length;
+
+      expect(numberLvl1Buttons).toEqual(1);
+      expect(numberLvl3Buttons).toEqual(3);
+    });
+
+    it('should keep the attention level on the first button with attention level 1', () => {
+      expect(buttons.findIndex((button) => button.isAttentionLevel1)).toEqual(0);
+    });
+
+    it('should enforce attention level rules when a button is added', () => {
+      // TODO: implement test...
+    });
+
+    it('should enforce attention level rules when a button is removed', () => {
+      // TODO: implement test...
```

Comment:
I could use some help on how to go about this - I've been searching for other tests that do something similar, but can't seem to find any. Any help would be much appreciated 🥳